packages/api/tests/controllers/nomination.test.js

```diff
@@ -13,6 +13,7 @@ const nominationData = {
   hospitalCity: 'Gotham',
   hospitalState: 'New Jersey',
   hospitalZipCode: '07320',
+  emailValidated: true,
```

Comment:
it would be pretty awesome if you guys were able to write a test that could show the email being sent